impl/src/main/java/org/ehcache/internal/store/heap/OnHeapStore.java

```diff
@@ -124,6 +124,8 @@ public void cacheConfigurationChange(CacheConfigurationChangeEvent event) {
     }
   };
 
+  private final OperationObserver<StoreOperationOutcomes.AccessOutcome> accessObserver = operation(StoreOperationOutcomes.AccessOutcome.class).named("access").of(this).tag("onheap-store").build();
+  private final OperationObserver<StoreOperationOutcomes.TierEfficiencyOutcome> tierEfficiencyObserver = operation(StoreOperationOutcomes.TierEfficiencyOutcome.class).named("faulting").of(this).tag("onheap-store").build();
```

Comment:
Why is the TierEfficiencyOutcome which we have tried to abstract from the implementation called "faulting"?